File: Server/src/main/java/org/f0w/fproject/server/service/extractor/ExtractionException.java
package org.f0w.fproject.server.service.extractor;

public class ExtractionException extends RuntimeException {
    public ExtractionException() {
        super();
    }

    public ExtractionException(String message) {
        super(message);
    }

    public ExtractionException(String message, Throwable cause) {
        super(message, cause);
    }

    public ExtractionException(Throwable cause) {
        super(cause);
    }

    protected ExtractionException(String message, Throwable cause, boolean enableSuppression, boolean writableStackTrace) {
        super(message, cause, enableSuppression, writableStackTrace);
    }
}

File: Server/src/main/java/org/f0w/fproject/server/service/extractor/ExtractionException.kt
package org.f0w.fproject.server.service.extractor

class ExtractionException : RuntimeException {
    constructor() : super() {}

    constructor(message: String) : super(message) {}

    constructor(message: String, cause: Throwable) : super(message, cause) {}

    constructor(cause: Throwable) : super(cause) {}

    protected constructor(message: String, cause: Throwable, enableSuppression: Boolean, writableStackTrace: Boolean)
        : super(message, cause, enableSuppression, writableStackTrace) {}
}
